linux_docker_resources/Dockerfile-RHEL

```
@@ -61,10 +64,10 @@ RUN dnf update \
 RUN rosdep init && rosdep update
 
 # Install a newer version of pytest
-RUN if test ${EL_RELEASE} = 8; then python3 -m pip install -U pytest pytest-rerunfailures; fi
+RUN if test ${EL_RELEASE/.*/} = 8; then python3 -m pip install -U pytest pytest-rerunfailures; fi
```

These could change to a removal instead of a replacement if anyone feels strongly.

```suggestion
RUN if test ${EL_RELEASE%%.*} = 8; then python3 -m pip install -U pytest pytest-rerunfailures; fi
```

---

I don't have a preference. My bigger concern was whether these bash-specific substitutions would work in the shell used by the container script but at least for almalinux 8 and 9 and Ubuntu Jammy, bash is used so this works.

I don't think we need to replace this with POSIX commands but I do think there's a risk here if bash ever stops being the default shell.

---

If I stumble across any more portable ways to make this happen, I'll make a note to follow-up.